www/templates/diary/infinite_scroll_list.html

```diff
@@ -48,43 +48,52 @@ <h3 ng-if="!infScrollControl.reachedEnd">{{'diary.scroll-to-load-more' | transla
-             <div class="row">
-                 <div class="col-90" ng-click="showDetail($event, trip)">
-                     <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
-                    <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #33e0bb;"></i>
-                            {{trip.start_display_name}}
-
+                    <div class="row">
+                        <div ng-click="showDetail($event, trip)" class="col-90 center-vert" ng-class="listTextClass" style="font-size: 14px; padding-left: 30px; margin-bottom: 0; justify-self: flex-start;" translate=".date-distance-in-time" translate-value-date="{{ trip.display_date }}" translate-value-distance="{{ trip.display_distance }}" translate-value-time="{{ trip.display_time }}"></div>
+                        <div ng-click="showDetail($event, trip)" class="col-10 diary-more-container center-vert center-horiz">
+                            <i class="ion-more"></i>
+                        </div>
                     </div>
-                    <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
-                    <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #ff5251;"></i>
-                            {{trip.end_display_name}}
+                    <div class="row">
+                        <div class="col-90" ng-click="showDetail($event, trip)">
+                            <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
+                            <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #33e0bb;"></i>
+                                    {{trip.start_display_name}}
+                            </div>
+                            <div ng-class="listLocationClass" id="no-border" href="#" style="background-color: transparent; font-size: 0.8em; padding-top: 5px; padding-bottom: 5px; padding-left: 30px; margin-top: 0; margin-bottom: 0;">
+                            <i class="icon ion-ios-location" style="font-size: 16px; left: 0; color: #ff5251;"></i>
+                                    {{trip.end_display_name}}
+                            </div>
+                        </div>
+                        <div ng-click="verifyTrip($event, trip)" class="col-10 diary-checkmark-container center-vert center-horiz">
+                            <i ng-class="trip.verifiability" class="ion-checkmark-round"></i>
```

Comment:
FYI, please continue using properties instead of functions here. Using functions to retrieve the `ng-class`, although it works, plays havoc with the caching and the performance really suffers.